docs/source/user/build_manually.rst

```diff
@@ -412,6 +496,10 @@ top of the file to the root build directory path.
 Nalu-Wind
 ~~~~~~~~~
 
+.. note::
+
+     The scripts referred to in the following section are available in the Nalu-Wind git history but have since been removed.
+
```

Comment:
Should we add a link to [do-config-nalu-wind-peregrine.sh](https://github.com/Exawind/build-test/blob/master/configs/machines/peregrine/do-config-nalu-wind-peregrine.sh) as a template that users should use until you clean up this section of docs?